config/site_settings.yml

```diff
@@ -2705,6 +2705,10 @@ uncategorized:
     default: 50000
     hidden: true
 
+  max_api_invites:
+      default: 50000
```

Comment:
Thoughts on reducing this to something way smaller? Like 100 or 500 max? I'm worried about how long the web request will take to process especially if email sending is on. I'd prefer users of this endpoint make multiple requests in batches this way they we get built in rate limiting with our max requests per min and such.

Reply:
yes, agreed. 50K definitely too high. customer is going to be using this endpoint to generate 300-400K invites however. Was thinking maybe 5K? I could write a test, see how long that takes to generate.

Reply:
I'm fine with 5k if the test goes smoothly. I just wanted a sane default, but we can always change the setting value for that specific customer.

Reply:
see test, took 65 seconds to execute ... that work?

Reply:
Thank you for making that test, but I don't think it's good to have http requests that take that long. We also wouldn't want to keep this long test in the test suite, but it is helpful for performance testing. I'm going to make an internal topic where we can discuss options because I'm not comfortable shipping this as is.